swiftUIHW/swiftUIHW/ContentView.swift

```diff
@@ -7,16 +7,120 @@
 
 import SwiftUI
 
+/// Экран покупки автомобиля
 struct ContentView: View {
+    private enum Constants {
+        static let carImageFrameWidthValue = 400.0
+        static let carImageFrameHeightValue = 500.0
+        static let clearString = ""
+        static let carColorText = "Цвет"
+        static let carColorLabelText = "Цвет автомобиля"
+        static let carDescriptionText = "Описание"
+        static let rubles = " рублей"
+        static let horsePower = " лошадиных сил"
+        static let carInformation = "Информацией об авто"
+        static let downloadText = "Скачать"
+        static let downloadedInfoText = "Пользователь скачал данные об авто"
+        static let shareText = "Поделиться"
+        static let bestCar = "BMW"
+    }
+    
+    @State private var segmentIndex = 0
+    @State private var colorSection = 0
+    @State private var isThirdShown = false
+    @State private var isSecondShown = false
+    
+    private var models = ["X5", "M5", "320i", "320d"]
+    private var prices = ["13 500 000", "19 270 315", "3 930 000", "3 600 000"]
+    private var hp = ["340", "625", "184", "184"]
+    private var colors = ["⚫️", "⚪️", "🟡", "🔴"]
+    
     var body: some View {
+        NavigationView {
+            Form {
+                Section {
+                    createCarImageSectionView()
+                }
+                Section(header: Text(Constants.carColorText)) {
+                    createCarColorSectionView()
+                }
+                Section(header: Text(Constants.carDescriptionText)) {
+                    createCarStatsSectionView()
+                }
+                Section {
+                    createButtonAlertsView()
+                }
+            }.navigationTitle("\(Constants.bestCar) \(models[segmentIndex])")
+            
+        }
+    }
+    
+    fileprivate func createCarImageSectionView() -> some View {
```

Comment:
```suggestion
    private var carImageSectionView: some View {
```
Ниже у всех поправь так же